source/includes/fact-environments.rst

```diff
@@ -0,0 +1,7 @@
+- `MongoDB Atlas
+  <https://www.mongodb.com/docs/atlas>`__: The fully
+  managed service for MongoDB deployments in the cloud
+- :ref:`MongoDB Enterprise <install-mdb-enterprise>`: The
+  subscription-based, self-managed version of MongoDB
+- :ref:`MongoDB Community <install-mdb-community-edition>`: The
+  source available, free-to-use, and self-managed version of MongoDB
```

Comment:
I: "source available" is a [compound modifier](https://www.mongodb.com/docs/meta/style-guide/style/punctuation/#hyphens-in-compound-modifiers) and should be hyphenated

```suggestion
  source-available, free-to-use, and self-managed version of MongoDB
```